internal/auth0/action/resource_trigger_action.go

```diff
@@ -111,6 +118,59 @@ func createTriggerAction(_ context.Context, d *schema.ResourceData, m interface{
 	return nil
 }
 
+func updateTriggerAction(_ context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	api := m.(*config.Config).GetAPI()
+
+	trigger := d.Get("trigger").(string)
+	actionID := d.Get("action_id").(string)
+	displayName := d.Get("display_name").(string)
+
+	currentBindings, err := api.Action.Bindings(trigger)
+	if err != nil {
+		if mErr, ok := err.(management.Error); ok && mErr.Status() == http.StatusNotFound {
+			d.SetId("")
+			return nil
+		}
```

Comment:
We shouldn't ignore the 404 here.

Reply:
Can you be more specific? This is consistent across the entire codebase.

Reply:
It's consistent across the entire codebase for resources that get created and get deleted. The set of trigger bindings is static and is not something a customer can create or delete.